src/test/scala/no/ndla/conceptapi/service/search/DraftConceptSearchServiceTest.scala

```diff
@@ -228,35 +228,39 @@ class DraftConceptSearchServiceTest extends IntegrationSuite(EnableElasticsearch
 
   test("That all returns all documents ordered by title ascending") {
     val Success(results) =
-      draftConceptSearchService.all(searchSettings.copy(sort = Sort.ByTitleAsc))
+      draftConceptSearchService.all(searchSettings.copy(sort = Sort.ByTitleAsc, pageSize = 20, fallback = true))
```

Comment:
Testene er oppdatert til at fallback er true og dermed returneres også id 11. Denne har en tittel som begynner på "e" og dermed plasserer den seg midt i resultatlista. Pagesize er med for å få med alle treff i resultatet.